adi/sync_start.py

```diff
@@ -55,6 +55,50 @@ def tx_sync_start_available(self):
         except:  # noqa: E722
             return "arm"
 
+    @property
```

Comment:
The B side methods won't be usable with non-Apollo ICs. Can we split these into a separate class then use inheritance to get the feature over to the AD9084 class. So look like:

```python

class sync_start_b(sync_start): # New class
    def tx_b_sync_start(self):
        ...

class ad9084(rx_tx, context_manager, sync_start_b): # updated
    ...



```